src/test/recovery/t/139_archive_mode_always.pl

```diff
@@ -0,0 +1,82 @@
+# test wal persistance for archive_mode 'always'
+use strict;
+use warnings;
+use PostgreSQL::Test::Cluster;
+use PostgreSQL::Test::Utils;
```

Comment:
The test works without this line. Is the line redundant?

Reply:
removed